app/components/e-heading.vue

```diff
@@ -0,0 +1,164 @@
+<!-- This component has no <template> because of dynamic root element -->
+
+<script>
+  export default {
+    name: 'e-heading',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the color of the heading (h1-h6)
+       */
+      color: {
+        type: String,
+        default: 'blue',
+        validator(value) {
+          return [
+            'blue',
+            'green',
+            'darkgrey',
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines the level of the heading (h1-h6)
+       */
+      level: {
+        type: String,
+        required: true,
+      },
+
+      /**
+       * Defines if the heading should have an underline
+       */
+      underline: {
+        type: Boolean,
+        default: null,
+      },
+
+      /**
+       * Defines the font weight
+       */
+      weight: {
+        type: String,
+        default: null,
+        validator(value) {
+          return [
+            'normal',
+            'semibold',
+            'bold',
+          ].includes(value);
+        },
+      },
+    },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+
+    /**
+     * Creates a heading element
+     *
+     * @param   {function}    createElement   Vue helper
+     *
+     * @returns {*}
+     */
+    render(createElement) {
+      const element = this.$props.level;
+      const attributes = {
+        class: this.b({
+          color: this.$props.color,
+          underline: this.$props.underline === 'true',
+          weight: this.$props.weight,
+        }),
+        attrs: {
+
+        },
+      };
+
+      return createElement(
+        element,
+        attributes,
+        this.$slots.default,
+      );
+    },
+  };
+</script>
+
+<style lang="scss">
+  .e-heading {
+    font-family: "Source Sans Pro", sans-serif;
+    font-weight: normal;
+    color: $color-secondary--1;
+    letter-spacing: 2.03px;
```

Comment:
Where is this coming from?

Reply:
Das kommt von: https://projects.invisionapp.com/d/main#/console/13587539/292245532/inspect :)